config/prometheus/rbac.yaml

```diff
@@ -0,0 +1,31 @@
+# creates Role and RoleBinding for prometheus-k8s service account to access our namespace
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: prometheus-k8s
+  namespace: system
```

Comment:
does it _have_ to be in this namespace with this name? these are generic names, I'm worried about conflicts with other components

Reply:
This is going to be deployed in the numaresources-operator namespace, as far as I understand.
We use "system" in the namespace field for other manifests too.

Reply:
ok, this is silly but still let's change the default to something like `numaresources` (not `system`). Plus, do we have evidence this goes in our namespace? a copy paste from the patched operator running on any cluster is good enough evidence.

Reply:
I'll add a screenshot later.
But what I'm saying is that "system" has been used across all the repo as a placeholder (look at manager.yaml e.g). This however replaced with "numaresources" after `make deploy`. kustomize replaces it.

Reply:
ok, this addresses my concerns, thanks